### DISCUSSION TITLE
install SCILAB-6.1.1 failed,Error: SHA256 mismatch

### DISCUSSION THREAD

**Discussion by bruce-the-terrible** [Homebrew/discussions#2147] [Everyday usage]
Error: SHA256 mismatch
Expected: 9e2c18ac6391ce43b068a997fa5dd959a62c3d02fc2499e018c0e88c1aff8eaf
 Actual: 922e879979fe4e1fde83be4b3df02070c0930a56a75cdeb9b5ef46ae29f0ef57

**Comment by SMillerDev** [comment]
What command are you running? It's hard to reproduce anything without reproducible steps.

**Comment by bruce-the-terrible** [comment]
MacBook-Pro ~ % brew install --cask scilab

or

MacBook-Pro ~ % brew install scilab


Same result


Sean Molenaar ***@***.***> 于 2021年9月17日周五 02:15写道：

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/111470

**Comment by a-a-maly** [comment]
Now the same for version 2024.1.0-x86_64. Probably, the author re-uploaded a newer version of the .dmg file.

~ % brew upgrade
==> Downloading https://formulae.brew.sh/api/formula.jws.json

==> Downloading https://formulae.brew.sh/api/cask.jws.json
###################################################################################################################################################### 100.0%
==> Casks with 'auto_updates true' or 'version :latest' will not be upgraded; pass `--greedy` to upgrade them.
==> Upgrading 1 outdated package:
scilab 2024.0.0 -> 2024.1.0
==> Upgrading scilab
==> Caveats
scilab requires Java 8. You can install it with:
  brew install --cask temurin@8

==> Downloading https://www.scilab.org/download/2024.1.0/scilab-2024.1.0-x86_64.dmg
==> Downloading from https://oos.eu-west-2.outscale.com/scilab-releases/2024.1.0/scilab-2024.1.0-x86_64.dmg
###################################################################################################################################################### 100.0%
==> Purging files for version 2024.1.0 of Cask scilab
Error: scilab: SHA256 mismatch
Expected: 382c38c43510ef12a9375c4c9ec777863ab9c241cffbac4e753596f35378e69c
  Actual: a856a62a85622654c6dff0a4035c1b9f152d1c6d99b4edee67e1589baa68c196

**Reply by miccal** [reply]
https://github.com/Homebrew/homebrew-cask/pull/176461
